packages/firestore/src/util/log.ts

```diff
@@ -15,58 +15,27 @@
  * limitations under the License.
  */
 
-import { Logger, LogLevel as FirebaseLogLevel } from '@firebase/logger';
+import { Logger, LogLevel } from '@firebase/logger';
 import { SDK_VERSION } from '../core/version';
 import { PlatformSupport } from '../platform/platform';
 
-const logClient = new Logger('@firebase/firestore');
+export const DEBUG = LogLevel.DEBUG;
+export const ERROR = LogLevel.ERROR;
+export const SILENT = LogLevel.SILENT;
 
-export const enum LogLevel {
-  DEBUG,
-  ERROR,
-  SILENT
-}
+const logClient = new Logger('@firebase/firestore');
 
 // Helper methods are needed because variables can't be exported as read/write
 export function getLogLevel(): LogLevel {
-  if (logClient.logLevel === FirebaseLogLevel.DEBUG) {
-    return LogLevel.DEBUG;
-  } else if (logClient.logLevel === FirebaseLogLevel.SILENT) {
-    return LogLevel.SILENT;
-  } else {
-    return LogLevel.ERROR;
-  }
-}
-export function setLogLevel(newLevel: LogLevel): void {
-  /**
-   * Map the new log level to the associated Firebase Log Level
-   */
-  switch (newLevel) {
-    case LogLevel.DEBUG:
-      logClient.logLevel = FirebaseLogLevel.DEBUG;
-      break;
-    case LogLevel.ERROR:
-      logClient.logLevel = FirebaseLogLevel.ERROR;
-      break;
-    case LogLevel.SILENT:
-      logClient.logLevel = FirebaseLogLevel.SILENT;
-      break;
-    default:
-      logClient.error(
-        `Firestore (${SDK_VERSION}): Invalid value passed to \`setLogLevel\``
-      );
-  }
+  return logClient.logLevel;
 }
 
-export function debug(tag: string, msg: string, ...obj: unknown[]): void {
```

Comment:
This was better as two separate functions--I don't think importing `DEBUG` is any better than importing `debug`.

I do agree that we should prefer not to import `debug` and `error` but I think a better way would be to just rename these to `logDebug` and `logError`. This would also serve to match iOS (which has `LOG_DEBUG` and `LOG_ERROR` macros) and still loosely match Android (which has `log.debug` and `log.error`).

This means that in the common case there's only one thing to import from here as well, where now every call site needs two.